content/support.html

```diff
@@ -22,7 +22,7 @@ <h2>Community Support for NATS and Clients</h2>
     <p>Often the quickest way to get help is to start a discussion with other NATS users. There
       are <a href="/documentation/contributing#communityhelp">several public forums</a> where you can discuss NATS with other knowledgeable users.</p>
     <h2>Commercial Support and Services</h2>
-    <p>Our team has decades of experience in distributed systems messaging, and are here to help you implement and maximize the benefits of a lightweight messaging system in any environment. Support is available for the NATS Server, and the Synadia supported clients listed <a href="/download">here</a>.</p>
-    <p><a href="mailto:info@nats.io">Contact</a> us today to learn more about how we can help you get the most out of NATS in your environment.</p>
+    <p>The NATS team, under the stewardship of <a href="https://synadia.com" target="_blank">Synadia Communications, Inc.</a>, has decades of experience in distributed systems messaging and are here to help you implement and maximize the benefits of a lightweight messaging system in any environment. Support is available for the NATS Server, and the Synadia supported clients listed <a href="/download">here</a>.</p>
+    <p>Contact our brilliant team at <a href="mailto:info@synadia.com">Synadia</a> for professional support services to learn more about how we can help you get the most out of NATS in your environment.</p>
```

Comment:
I'd suggest "world class" over than brilliant.  Also, rephrase to something like "...for commercial support and professional services to help you best leverage NATS for your business."  wdyt?

Reply:
Sounds good - thanks!